components/kyma-environment-broker/internal/process/deprovisioning/initialisation_test.go

```diff
@@ -119,45 +120,19 @@ func TestInitialisationStep_Run(t *testing.T) {
 }
 
 func fixDeprovisioningOperation() internal.DeprovisioningOperation {
-	return internal.DeprovisioningOperation{
-		Operation: internal.Operation{
-			ID:                     fixOperationID,
-			InstanceID:             fixInstanceID,
-			ProvisionerOperationID: fixProvisionerOperationID,
-			ProvisioningParameters: internal.ProvisioningParameters{ErsContext: internal.ERSContext{UserID: "test"}},
-			CreatedAt:              time.Now(),
-			UpdatedAt:              time.Now(),
-		},
-	}
+	deprovisioniningOperation := fixture.FixDeprovisioningOperation(fixOperationID, fixInstanceID)
+	return deprovisioniningOperation
 }
 
 func fixProvisioningOperation() internal.ProvisioningOperation {
-	planID := broker.AzurePlanID
-
-	return internal.ProvisioningOperation{
-		Operation: internal.Operation{
-			ID:                     fixOperationID,
-			InstanceID:             fixInstanceID,
-			ProvisionerOperationID: fixProvisionerOperationID,
-			CreatedAt:              time.Now(),
-			UpdatedAt:              time.Now(),
-			ProvisioningParameters: internal.ProvisioningParameters{
-				ErsContext: internal.ERSContext{GlobalAccountID: "1"},
-				PlanID:     planID,
-			},
-		},
-	}
+	provisioningOperation := fixture.FixProvisioningOperation(fixOperationID, fixInstanceID)
+	return provisioningOperation
 }
 
 func fixInstanceRuntimeStatus() internal.Instance {
-	return internal.Instance{
-		InstanceID:      fixInstanceID,
-		RuntimeID:       fixRuntimeID,
-		DashboardURL:    "",
-		GlobalAccountID: fixGlobalAccountID,
-		CreatedAt:       time.Time{},
-		UpdatedAt:       time.Time{},
-		DeletedAt:       time.Time{},
-		ServicePlanID:   broker.AzurePlanID,
-	}
+	instance := fixture.FixInstance(fixInstanceID)
+	instance.RuntimeID = fixRuntimeID
```

Comment:
I think we should not overwrite those here, instead we should make sure the source of truth is fixture from `fixture` package and get rid of `const` in most of steps unit test like those `fixRuntimeID` and `fixGlobalAccountID` here.

Reply:
Brain storm session took place. Team agreed to use local constants.